Add area name translations to Local vs. Global

antw opened the pull request:
Moves area translations to a namespace so that they may be included in the JavaScript translations used by Local vs. Global.

This affects everywhere in the ETM where we show translated area names. I think I got them all, but I'd be grateful for a second pair of eye checking that everything looks correct.

Ref #3089

marliekeverweij commented:
I've checked the:
- CO2 factsheet
- Energymix factsheet
- ETM in general
- Local vs. Global
- Scenario report

I only found missing translations in the scenario report:
![Screen Shot 2019-09-24 at 10 04 24](https://user-images.githubusercontent.com/32833996/65493545-5cdcd780-deb3-11e9-8e11-2d73ed85825f.png)
![Screen Shot 2019-09-24 at 10 04 08](https://user-images.githubusercontent.com/32833996/65493548-5f3f3180-deb3-11e9-8638-063ce8c055d1.png)

I couldn't open the multi-year charts,  somehow it was not visible:
![Screen Shot 2019-09-24 at 10 06 08](https://user-images.githubusercontent.com/32833996/65493497-3e76dc00-deb3-11e9-9096-157d666f37d7.png)

antw commented:
I fixed the missing translation in the report title. The "1.1xAREA_CODE" is something we do with national scenarios (e.g. "1.1xNL"); we don't usually show the area code for smaller datasets because it doesn't look nice. I've changed the regional report to show just "1.1x", which is consistent with the dashboard.

I've checked the multi-year charts pages myself, so they'll be fine. They only show up in your local copy [if your config/config.yml has a `multi_year_charts_url` specified](https://github.com/quintel/etmodel/blob/89af47b6e6db00c6d3e61a80662d174b7d217f20/config/config.sample.yml#L11-L12).
